osu.Game.Rulesets.Tau/Mods/TauModHidden.cs

```diff
@@ -17,7 +19,7 @@ public class TauModHidden : ModHidden
         private const double fade_in_duration_multiplier = 0.4;
         private const double fade_out_duration_multiplier = 0.3;
 
-        public override bool HasImplementation => false;
+        protected override bool IsFirstAdjustableObject(HitObject hitObject) => !(hitObject is HardBeat);
```

Comment:
Is there a reason why we specifically ignore the existence of HardBeats, and never apply the increased visibility state to it?

Reply:
oh woops, i forgot to remove that when i was testing things